CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 17.7.0
+
+_Released 2025-08-22_
+
+This update lays groundwork for Search Pagination, which will allow Search steps to paginate through results so that the most relevant results can be returned. However, this is not yet supported by any Zapier products.
```

Comment:
I think I'd keep it generic and say "Search actions" since steps is Zap specific?